components/proposalSignForm.tsx

```diff
@@ -39,17 +44,62 @@ function ProposalSignForm({
     let cc = await state.connection.wallet.at(address);
     let versioned = VersionedApi(version, address);
 
-    await versioned.signProposal(
-      cc,
-      state.connection,
-      proposal,
-      result,
-      Boolean(resolve)
+    setTimeoutAndHash(
+      await versioned.signProposal(
+        cc,
+        state.connection,
+        proposal,
+        result,
+        Boolean(resolve)
+      )
+    );
+  }
+
+  if (timeoutAndHash[0]) {
+    return (
+      <div className="mx-auto mt-4 w-full text-center text-zinc-400">
+        <p>
+          The wallet {"didn't"} confirmed that the transaction has been
```

Comment:
```suggestion
          The wallet {"can't"} confirm that the transaction has been
```
would it be better :thinking: ? "didn't" sounds that the wallet even didn't give it a try?  so as the others.

Reply:
You're right, It sounds better